sensor/common/sensor/sensor.go

```diff
@@ -72,12 +72,17 @@ type Sensor struct {
 
 	stoppedSig concurrency.ErrorSignal
 
-	notifyList []common.Notifiable
-	reconnect  atomic.Bool
+	notifyList        []common.Notifiable
+	reconnect         atomic.Bool
+	hashReconciliator reconciliationHandler
+}
+
+type reconciliationHandler interface {
+	ProcessHashes(map[string]uint64) []string
```

Comment:
A better return type would be: `central.MsgFromSensor` which already contains the delete action. Each store should be able to generate a delete message for their own resource type.